### PR TITLE
Fixes #1162 - replaces about page with an Android Fragment

### LINKED ISSUE #1162: FNX-4839 ⁃ Long URLs cause Fenix to become unresponsive for tens of seconds
## Steps to reproduce
1. Visit `about:version` or tap Menu → Settings → About Fenix
(This is automatically replaced by a large `data:` URI.)
1. Tap on the URL bar

### Expected behavior
1. The URL bar pops up and is ready for input.

### Actual behavior
1. Fenix becomes unresponsive for 20 seconds. The URL bar and search suggestion pages _s l o w l y_ load into view, piece by piece.
This appears to get worse the more times I reproduce it.

### Device information
* Android device: Pixel (original)
* Fenix version: 1.0.1912 (Build #10811221 :lizard: 67.0a1-20190312095443)

### PR DESCRIPTION
**Pull request by boek**
May we never talk about this again :joy:

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -299,4 +312,7 @@
 
     <!-- Content Description for session item share button -->
     <string name="content_description_session_share">Share session</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (sblatz) [inline app/src/main/res/values/strings.xml]: What purpose is this serving?

Comment (boek) [reply app/src/main/res/values/strings.xml]: ...code gen :man_facepalming:

File: app/src/main/res/layout/fragment_about.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto" xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="org.mozilla.fenix.settings.AboutFragment">
+
+    <ImageView
+        android:id="@+id/wordmark"
+        android:contentDescription=""
```
Comment (colintheshots) [inline app/src/main/res/layout/fragment_about.xml]: Technically, it's better to use importantForAccessibility="no".

File: app/src/main/res/layout/fragment_about.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        android:src="?attr/fenixLogo"
+        android:layout_marginTop="24dp"
+        android:layout_height="wrap_content"
+        android:layout_width="0dp"
+        app:layout_constraintWidth_percent="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="ContentDescription" />
```
Comment (colintheshots) [inline app/src/main/res/layout/fragment_about.xml]: same here

**Pull request merged**